README.adoc

```diff
@@ -292,36 +292,11 @@ The compatibility of the specified version with Develocity can be found https://
-|2.8                            | 1.17.3                                             | 1.12                                                      | 2023.1
-|2.7                            | 1.17.2                                             | 1.11.1                                                    | 2023.1
-|2.6                            | 1.17.1                                             | 1.11.1                                                    | 2023.1
-|2.5.1                          | 1.17                                               | 1.11.1                                                    | 2023.1
-|2.5                            | 1.17                                               | 1.11.1                                                    | 2023.1
-|2.4                            | 1.16.6                                             | 1.11.1                                                    | 2022.4
-|2.3.2                          | 1.16.5                                             | 1.11.1                                                    | 2022.4
-|2.3.1                          | 1.16.5                                             | 1.11.1                                                    | 2022.4
-|2.3                            | 1.16.4                                             | 1.11.1                                                    | 2022.4
-|2.2                            | 1.16.1                                             | 1.11.1                                                    | 2022.4
-|2.1.1                          | 1.15.5                                             | 1.11.1                                                    | 2022.3
-|2.1                            | 1.15.4                                             | 1.11.1                                                    | 2022.3
-|2.0                            | 1.15.4                                             | 1.11.1                                                    | 2022.3
-|1.40                           | 1.15.3                                             | 1.11.1                                                    | 2022.3
-|< 1.40                         | *Not supported*                                    | *Not supported*                                           | *Not supported*
-|===
+For Maven builds the version used for the Develocity Maven extension is defined in the `Develocity Maven extension version` field in the `Maven settings` section of the configuration form.
+See <<Enable auto-injection>> for details.
+The compatibility of the specified version with Develocity can be found https://docs.gradle.com/enterprise/compatibility/#maven_extension[here].
+
+For the optional Common Custom User Data Maven extension which is defined the same form, you can see the compatibility of the specified version with the Develocity Maven extension https://github.com/gradle/common-custom-user-data-maven-extension#version-compatibility[here].
 
```

Comment:
Shouldn't we put a warning note explaining that the injection will not work after upgrading referencing https://github.com/jenkins-infra/helpdesk/issues/4263.

Reply:
I thought about adding it to the release notes only.
Do you think we should mention it in the README as well?

Reply:
Yes, seems like a breaking change that should stay, see for example what we did when requiring Develocity 2024.1 https://github.com/gradle/develocity-gitlab-templates?tab=readme-ov-file#requirements

Reply:
Agree, although I'd update a Readme at the time of release, as it might be confusing to have this mentioned already without a version available